{"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-gui/pullrequests/81/comments/53618924.json"}, "html": {"href": "#!/ignitionrobotics/ign-gui/pull-requests/81/_/diff#comment-53618924"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 81, "links": {"self": {"href": "data/repositories/ignitionrobotics/ign-gui/pullrequests/81.json"}, "html": {"href": "#!/ignitionrobotics/ign-gui/pull-requests/81"}}, "title": "Plot widget - part4 (last)"}, "content": {"raw": "I see the following error when I \"Clear all fields\" with data currently being plotted.\n\n```\n[GUI] [Err] [Plot.cc:398] Failed to find plot data\n```", "markup": "markdown", "html": "<p>I see the following error when I \"Clear all fields\" with data currently being plotted.</p>\n<div class=\"codehilite\"><pre><span></span>[GUI] [Err] [Plot.cc:398] Failed to find plot data\n</pre></div>", "type": "rendered"}, "created_on": "2018-01-12T22:41:30.128549+00:00", "user": {"display_name": "Nate Koenig", "uuid": "{c862cdd9-fcc8-4419-9fc7-e20db14b8fcb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D"}, "html": {"href": "https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:095b1e12-74ed-4e20-b44f-2f0745b616e0/ca24bb11-4787-4b14-a20d-d91835e9bde2/128"}}, "nickname": "Nathan Koenig", "type": "user", "account_id": "557058:095b1e12-74ed-4e20-b44f-2f0745b616e0"}, "updated_on": "2018-01-12T22:41:30.131031+00:00", "type": "pullrequest_comment", "id": 53618924}